risc0/zkp/src/prove/accum.rs

```diff
@@ -88,7 +95,9 @@ impl<'a, F: Field> CircuitStepHandler<F::Elem> for Handler<'a, F> {
     fn call(
         &mut self,
         cycle: usize,
+        // TODO what is name?
```

Comment:
This is the name of the extern call to perform.
Examples: `getMajor`, `ramRead`, `syscall`